app/src/main/java/org/mifos/selfserviceapp/ui/activities/LoginActivity.java

```diff
@@ -59,7 +59,7 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     public void onLoginSuccess(String userName) {
         loginStatus = true;
-        showToast(getString(R.string.toast_welcome, userName));
+        showToast(getString(R.string.toast_welcome, userName.replaceFirst("\\s++$", "")));
```

Comment:
Yeah, revert back because the Username which we are showing in Toast. It is server response, not user entered username.